cluster/manifests/03-ebs-csi/02-csi-driver.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: apps/v1
```

Comment:
Since we're just testing it, let's make it conditional? Deploying it to all clusters might not be feasible because we might not have enough space on the nodes.

Reply:
I will tie it to the same config item which enables the CSI backed PVCs for prometheus.